applications/advanced-concepts/locale.md

```diff
@@ -0,0 +1,35 @@
+---
+parent: Building and research
+title: Locale
+description: Specification of language variants
+---
+
+A **locale** is an identifier of a language and region plus an optional writing script.
+The locale is used in [machine translation APIs](/apis/apis.md) to specify the language of the source and target text.
+Locales are commonly used to set the default language on mobile phones and computers.
```

Comment:
I guess we can leave this info out as it may not be strictly MT-relevant.

Reply:
Yeah after thinking some more I agree. 

There might be a need somewhere for it but probably not here. What I was thinking is that systems will pull locale from the OS and send that to an MT API. Maybe Chrome works that way, I'm not sure. The other way around, some locales may not be utilized simply because users don't set locale to their preferred language. Likewise in MT-based localization of apps it uses those codes.